core/libs/aspect-exporter/src/lib/visitor/entity-value/entity-value-visitor.ts

```diff
@@ -42,22 +40,60 @@ export class EntityValueVisitor extends BaseVisitor<DefaultEntityValue> {
     const {aspectModelUrn} = entityValue;
     const rdfModel = this.modelService.currentRdfModel;
 
-    entityValue.properties.forEach(property => {
-      const object = this.createObjectForRDF(property);
-      const {key, value} = property;
+    const {propertyCollectionWithLangString, property} = entityValue.properties.reduce(
+      (acc, property) => (
+        isDataTypeLangString(property.key.property) && property.key.property.characteristic instanceof DefaultCollection
+          ? acc.propertyCollectionWithLangString.push(property)
+          : acc.property.push(property),
+        acc
+      ),
+      {
+        propertyCollectionWithLangString: [],
+        property: [],
+      }
+    );
+
+    if (propertyCollectionWithLangString.length > 0) {
+      const withoutEmptyLangString = propertyCollectionWithLangString.filter(prop => {
+        return prop.value.value !== null && prop.value.value !== undefined;
+      });
+
+      const langStringRdfObject = withoutEmptyLangString.map(this.createObjectForCollectionLangStringRDF.bind(this));
+      this.rdfListService.pushEntityValueLangString(entityValue, ...langStringRdfObject);
+    }
+
+    if (property.length > 0) {
+      const propertiesWithoutEmptyLangString = property.filter(prop => {
+        const urn = prop.key.property.characteristic.dataType?.getUrn();
+        return !(this.getDataTypeFromUrn(urn) === 'langString' && prop.value === '');
+      });
+
+      propertiesWithoutEmptyLangString.forEach(property => {
+        const rdfObject = this.createObjectForRDF(property);
+        const {key, value} = property;
 
-      rdfModel.store.addQuad(DataFactory.namedNode(aspectModelUrn), DataFactory.namedNode(key.property.aspectModelUrn), object);
+        rdfModel.store.addQuad(DataFactory.namedNode(aspectModelUrn), DataFactory.namedNode(key.property.aspectModelUrn), rdfObject);
 
-      this.handleExternalReference(value, aspectModelUrn);
-    });
+        this.handleExternalReference(value, aspectModelUrn);
+      });
+    }
+  }
+
+  private createObjectForCollectionLangStringRDF(ev: EntityValueProperty): {predicate: NamedNode; literal: Literal} {
+    const langString = ev.value as LangStringProperty;
+    const predicate = DataFactory.namedNode(ev.key.property.aspectModelUrn);
+    return {
+      predicate: predicate,
+      literal: DataFactory.literal(langString?.value?.toString(), langString?.language?.toString()),
+    };
   }
 
-  private createObjectForRDF({key, value}: EntityValueProperty): any {
+  private createObjectForRDF({key, value}: EntityValueProperty): NamedNode | Literal {
     if (value instanceof DefaultEntityValue) {
       return DataFactory.namedNode(value.aspectModelUrn);
     }
 
-    if (isDataTypeLangString(key.property)) {
+    if (isDataTypeLangString(key.property) && key.property.characteristic instanceof DefaultCharacteristic) {
```

Comment:
Here you make sure its not an Collection type -> List, Set, SortedSet ... remove "key.property.characteristic instanceof DefaultCharacteristic" and make sure its not an Collection Type with "! ... List, Set, SortedSet ..." Be sure you check all Collection types or that one that are extending it.